### PR TITLE
Add basic decoding and visualization of trip planning response

### PR DESCRIPTION
**Pull request by doggkruse**
Display via mapbox if MAPBOX_API_KEY is provided in .env

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +100,8 @@ bin/rivian_cli --poll
 
 ### Trip planning
 ```
-bin/rivian_cli --plan_trip 85,225,42.0772,-71.6303,42.1399,-71.5163
+bin/rivian_cli --plan_trip 85,225,40.5112,-89.0559,39.9310,-104.9530
```
Comment (the-mace) [inline README.md:133]: The first coordinate looks like Rivian, but the second is perhaps a personal address. I'd suggest against any personal addresses (or vins, etc) in examples/data.

File: src/rivian_python_api/rivian_map.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import os
+import json
+import math
+import polyline
```
Comment (the-mace) [inline src/rivian_python_api/rivian_map.py:4]: Needs to be added to requirements.txt

File: src/rivian_python_api/rivian_map.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import plotly.graph_objects as go
```
Comment (the-mace) [inline src/rivian_python_api/rivian_map.py:5]: Also needs to be in requirements.txt

**Review by the-mace — CHANGES_REQUESTED** [review]
Would you mind making the suggested changes?

File: README.md (hunk 1 above)
Comment (doggkruse) [reply README.md:133]: I agree!  The second coordinate is the Rivian Service center in Colorado so should be OK as is

File: src/rivian_python_api/rivian_map.py (hunk 2 above)
Comment (doggkruse) [reply src/rivian_python_api/rivian_map.py:4]: requirements.txt does not exist in the repo (maybe .gitignore?) would you like me to create it or if you'd like to add the existing I can update.

File: src/rivian_python_api/rivian_map.py (hunk 3 above)
Comment (doggkruse) [reply src/rivian_python_api/rivian_map.py:5]: Same as above

**Comment by doggkruse** [comment]
Added requirements.txt

**Comment by the-mace** [comment]
Nice, thank you!

**Pull request merged**